asabiris/orchestration/sendmail.py

```diff
@@ -53,7 +52,7 @@ async def send_mail(
 		# Render a body
 		body_html, email_subject_body = await self.render(body_template, body_params)
 
-		if email_subject is None:
```

Comment:
This line was right.
If Subject is not in the request, use the one from template.
If Subject is not in the template, use the default from the configuration.

Reply:
@eliska-n What happens if subject is in the request and template?

Reply:
@eliska-n is the order of picking up the subject "request > template > config?